plugins/action_trim_responses/src/trimResponses.ts

```diff
@@ -52,9 +52,12 @@ const trimResponses: ActionPluginType = async ({ parameters, applicationData, DB
       const previousResponse =
         responseArray.length > 1 ? responseArray[responseArray.length - 2] : null
       if (
-        isEqual(latestResponse.value, previousResponse?.value) ||
-        latestResponse.value === null || // Application Responses
-        latestResponse.value?.decision === null // Review responses
+        (previousResponse !== null && isEqual(latestResponse.value, previousResponse?.value)) ||
+        // Application Responses
+        (latestResponse.value === null &&
+          latestResponse.is_reviewable !== IsReviewableStatus.Always) ||
```

Comment:
This is the only real bit of additional logic -- if the element is marked as Always, then we don't trim it.